src/get-started/production/index.md.njk

```diff
@@ -33,7 +33,7 @@ Using this option, you will be able to:
 - selectively [include the CSS or JavaScript](https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/) for individual components
 - build your own styles or components based on the palette or typography and spacing mixins
 - customise the build (for example, overriding colours or enabling global styles)
-- use the component Nunjucks templates
+- use the [component Nunjucks macros](https://frontend.design-system.service.gov.uk/use-nunjucks-macros/) if your environment supports them
```

Comment:
Related to [the suggestion on the Frontend docs PR](https://github.com/alphagov/govuk-frontend-docs/pull/67/files/96d2cd59218ae9476ee6591263da2c47bed8c989#diff-2772ad00093008ab328b992600657eff), I wonder if this should mention both the page template and the component macros.

Reply:
Thanks - tweaked!